index.php

```diff
@@ -251,9 +250,11 @@
 		$player->global_message_viewed = 1;
 	}
 	if(!$player->global_message_viewed && !$ajax) {
-		$result = $system->query("SELECT `global_message` FROM `system_storage` LIMIT 1");
-		$message = $system->db_fetch($result)['global_message'];
+		$result = $system->query("SELECT `global_message`, `time` FROM `system_storage` LIMIT 1");
+		$results = $system->db_fetch($result, 'assoc');
```

Comment:
```suggestion
		$results = $system->db_fetch($result);
```
`'assoc'` is already the default so we don't need to specify it